lib/controllers/github-tab-controller.js

```diff
@@ -6,23 +6,23 @@ import yubikiri from 'yubikiri';
 import RemotePrController from './remote-pr-controller';
 import GithubLoginModel from '../models/github-login-model';
 import ObserveModel from '../views/observe-model';
-import {RemotePropType} from '../prop-types';
+import {RemotePropType, BranchPropType} from '../prop-types';
```

Comment:
question: do we have any conventions around import ordering or is it YOLO / free for all?

Reply:
ehh, I find eslint's ordering import rule to be really weird and arbitrary and hard to remember. 
https://eslint.org/docs/rules/sort-imports

I'm ok with yolo tbh! Or with a like "try to alphabetize things but nbd if you don't"